cloudant-client/src/test/java/com/cloudant/tests/ViewsTest.java

```diff
@@ -623,19 +622,14 @@ public void scalarValues() throws Exception {
     @Test
     public void viewWithNoResult_emptyList() throws IOException {
         init();
-        assertEquals("The results list should be of length 0", 0, db.getViewRequestBuilder
-                ("example", "by_tag").newRequest(Key.Type.STRING, Object.class).keys
-                ("javax").build().getResponse().getKeys().size());
+        assertEquals(0, db.getViewRequestBuilder("example", "by_tag").newRequest(Key.Type.STRING, Object.class).keys("javax").build().getResponse().getKeys().size(), "The results list should be of length 0");
```

Comment:
It seems like some of these code replacements have produced very long lines. Perhaps at the end of the review process it is just worth applying the formatter to all the classes before the final merge?

Reply:
Good idea, i've added it to the TODOs